strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/layout/npm/NpmArtifactController.java

```diff
@@ -82,45 +95,187 @@
     @Qualifier("npmJackasonMapper")
     private ObjectMapper npmJackasonMapper;
 
-    @RequestMapping(path = { "{storageId}/{repositoryId}/npm"}, method = RequestMethod.GET)
+    @Inject
+    private RepositoryProviderRegistry repositoryProviderRegistry;
+
+    @Inject
+    private NpmPackageSupplier npmPackageSupplier;
+    
+    @RequestMapping(path = { "{storageId}/{repositoryId}/npm" }, method = RequestMethod.GET)
     public ResponseEntity<String> greet()
     {
-        //TODO: find out what NPM expect to receive here
+        // TODO: find out what NPM expect to receive here
         return ResponseEntity.ok("");
```

Comment:
I don't think we need such a method. This was introduces by Alex back in the day for testing purposes. I think we could just remove this all-together.

Reply:
this method called by NPM client, so it's not just for testing

Reply:
Interesting! Thanks for pointing this out!